CONTRIBUTING.md

```diff
@@ -22,7 +22,12 @@ Table of contents:
 
 ### Setup local dev environment
 
-To run and develop the module NodeJS 16 is required. In this section 2 ways of configuring a development environment are described.
+To run and develop the module NodeJS 16.13.0 is required. 
```

Comment:
Is it really that exactly 16.13.0 is required? I would expect that it works with any v16?